net.zdechov.app.Tunneler.yml

```diff
@@ -0,0 +1,47 @@
+app-id: net.zdechov.app.Tunneler
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.freepascal
+command: Tunneler
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --filesystem=home
```

Comment:
why does it need home permissions?

Reply:
It stores its settings in ~/.config/Tunneler but I guess no. Removed.

Reply:
you could use `--persist` to make sure it stays. Unless it's actually using XDG directories which mean it's handled.

Reply:
From the testing it looks it remembers the settings. So it is ok as is.